docs/pages/2019_IIT_Mandi_India/sections/first_taste.rst

```diff
@@ -133,12 +134,16 @@ A few questions you could answer using these commands (optional)
  * How many SCF iterations were needed for convergence?
  * How long did Quantum ESPRESSO actually run (wall time)?
 
+.. tip::
+
+    Use the ``grep`` command to filter the terminal output by keywords, e.g., ``verdi calc job res 175 | grep wall_time``.
+
 Moving to a different computer
 ------------------------------
 
-Now, this Quantum ESPRESSO calculation ran on your (virtual) machine.
-This is fine for tests, but for production calculations you'll typically want to run on a remote compute cluster.
-In AiiDA, moving a calculation from one computer to another means changing one line of code.
+The Quantum ESPRESSO calculation we just ran, was directly executed on the virtual machine.
```

Comment:
I have a feeling that "executed" is not the right word, although I may be wrong.
It's coming from the fact that execution may mean "start the calculation", i.e., send it to the daemon.
While what is meant here, is that the calculation was performed using the VM's "hardware".